packages/@aws-cdk/aws-efs/test/integ.efs.ts

```diff
@@ -9,6 +9,7 @@ const vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 3, natGateways: 1 });
 
 const fileSystem = new FileSystem(stack, 'FileSystem', {
   vpc,
+  vpcSubnets: vpc.selectSubnets(undefined),
```

Comment:
This doesn't seem right.

1. Why is this needed? the `FileSystem` construct will already call `selectSubnets` with `undefined`.
2. The `selectSubnets` method returns `SelectedSubnets`, and not `SubnetSelection`, those two interfaces are not meant to be interchangeable, the fact they currently pass structural typing is somewhat accidental.

I'm pretty sure we can drop this change.